src/backend/marsha/core/tests/management_commands/test_sync_medialive_video.py

```diff
@@ -0,0 +1,407 @@
+                },
+                {
+                    "Name": f"test_{live2_id}_1667490961",
+                    "Tags": {"environment": "test"},
+                    "State": "Creating",
+                },
+                {
+                    "Name": f"test_{live3_id}_1667490961",
+                    "Tags": {"environment": "test"},
+                    "State": "Recovering",
+                },
+                {
+                    "Name": f"test_{live4_id}_1667490961",
+                    "Tags": {"environment": "test"},
+                    "State": "Updating",
+                },
+            ]
+
+            call_command("sync_medialive_video", stdout=out)
+
+            self.assertIn(f"Checking video {live1_id}", out.getvalue())
+            self.assertIn(f"Checking video {live2_id}", out.getvalue())
+            self.assertIn(f"Checking video {live3_id}", out.getvalue())
+            self.assertIn(f"Checking video {live4_id}", out.getvalue())
+
+            live1.refresh_from_db()
+
+            self.assertEqual(live1.live_state, RUNNING)
+            self.assertEqual(live2.live_state, RUNNING)
+            self.assertEqual(live3.live_state, RUNNING)
+            self.assertEqual(live4.live_state, RUNNING)
+
+        out.close()
+
+    def test_sync_idle_medialive_on_stopped_video(self):
+        """
+        Video state STOPPED is equivalent to medialive channel state IDLE
+        and should do nothing.
+        """
+        # stopped and idle lives should not change.
+        live1_id = uuid.uuid4()
+        live1 = self.createLive(live1_id, STOPPED)
+        live2_id = uuid.uuid4()
+        live2 = self.createLive(live2_id, IDLE)
+
+        # Run command
+        out = StringIO()
+        with mock.patch(
+            "django.utils.timezone.now",
+            return_value=datetime(2022, 10, 14, 15, 25, tzinfo=timezone.utc),
+        ), Stubber(
+            medialive_utils.medialive_client
+        ) as mediapackage_client_stubber, mock.patch.object(
+            sync_medialive_video, "list_medialive_channels"
+        ) as list_medialive_channels_mock:
+            mediapackage_client_stubber.add_response(
+                "batch_update_schedule",
+                service_response={},
+            )
+            list_medialive_channels_mock.return_value = [
+                {
+                    "Name": f"test_{live1_id}_1667490961",
+                    "Tags": {"environment": "test"},
+                    "State": "Idle",
+                },
+                {
+                    "Name": f"test_{live2_id}_1667490961",
+                    "Tags": {"environment": "test"},
+                    "State": "Idle",
+                },
+            ]
+
+            call_command("sync_medialive_video", stdout=out)
+
+            self.assertIn(f"Checking video {live1_id}", out.getvalue())
+            self.assertIn(f"Checking video {live2_id}", out.getvalue())
+
+            live1.refresh_from_db()
```

Comment:
why not refreshing `live2` ? you are testing it after, you don't know if it has change or not.